File: app/src/main/java/com/github/kongpf8848/animation/activity/LottieAnimActivity.java
package com.github.kongpf8848.animation.activity;

import android.view.View;

import com.github.kongpf8848.animation.R;
import com.github.kongpf8848.animation.activity.lottie.AutoHomeActivity;
import com.github.kongpf8848.animation.activity.lottie.BossActivity;
import com.github.kongpf8848.animation.activity.lottie.CardActivity;

import butterknife.OnClick;

public class LottieAnimActivity extends BaseActivity {

    @Override
    protected int getLayoutId() {
        return R.layout.activity_lottie;
    }

    @OnClick(R.id.button1)
    public void onButton1(View view) {
        startActivity(AutoHomeActivity.class);
    }

    @OnClick(R.id.button2)
    public void onButton2(View view) {
        startActivity(BossActivity.class);
    }

    @OnClick(R.id.button3)
    public void onButton3(View view) {
        startActivity(CardActivity.class);
    }
}

File: app/src/main/java/com/github/kongpf8848/animation/activity/LottieAnimActivity.kt
package com.github.kongpf8848.animation.activity

import android.view.View
import butterknife.OnClick
import com.github.kongpf8848.animation.R
import com.github.kongpf8848.animation.activity.lottie.AutoHomeActivity
import com.github.kongpf8848.animation.activity.lottie.BossActivity
import com.github.kongpf8848.animation.activity.lottie.CardActivity
import com.github.kongpf8848.animation.base.BaseToolbarActivity

class LottieAnimActivity : BaseToolbarActivity() {

    override fun getLayoutId(): Int {
        return R.layout.activity_lottie
    }

    @OnClick(R.id.button1)
    fun onButton1(view: View?) {
        startActivity(AutoHomeActivity::class.java)
    }

    @OnClick(R.id.button2)
    fun onButton2(view: View?) {
        startActivity(BossActivity::class.java)
    }

    @OnClick(R.id.button3)
    fun onButton3(view: View?) {
        startActivity(CardActivity::class.java)
    }
}